db/migrate/20190226022413_add_deleted_at_to_models.rb

```diff
@@ -0,0 +1,11 @@
+class AddDeletedAtToModels < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :deleted_at, :datetime
+    add_column :assets, :deleted_at, :datetime
+    add_column :tracks, :deleted_at, :datetime
+    add_column :playlists, :deleted_at, :datetime
+    add_column :listens, :deleted_at, :datetime
+    add_column :comments, :deleted_at, :datetime
+    add_column :posts, :deleted_at, :datetime
```

Comment:
Don't hate me (or do!) but along with `Topic` we can also ignore `Post` — both of these are for the old forum. Although it might be nice to delete stuff from the new forum eventually, we can figure that out some other time...

Reply:
+1